### DISCUSSION TITLE
I'm creating a new project i need a dev to help me

### DISCUSSION THREAD

**Discussion by Shammann21** [community/community#144573] [Programming Help]
### Body

i have a great Dapp idea, i have been trying to dev it by my self but my low skill plus AI help is not enought. I need help from a solidity dev . don t hesitate to contact me !

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by CleverLemming1337** [comment]
Hi there 👋
Maybe I can help you if you could explain your idea further.

**Comment by yunwei-zh** [comment]
Hi Shammann,
Hope you are doing well
Please let me know about your ideas, I will help you with proven experience in developing DApps

**Comment by abdelillahCoder** [comment]
Your message looks great! It conveys enthusiasm and a clear request for collaboration. To make it even better, you might want to briefly mention what makes your DApp idea unique or exciting—this can help attract the right developer. For example:

Highlight the innovation: "My DApp idea focuses on [e.g., solving a specific problem, introducing a novel concept in blockchain]."
Add a personal touch: "I’m passionate about blockchain and eager to learn while collaborating."
This makes your message more engaging and appealing to potential contributors.

**Comment by iscordian** [comment]
> ### Body
> i have a great Dapp idea, i have been trying to dev it by my self but my low skill plus AI help is not enought. I need help from a solidity dev . don t hesitate to contact me !
> 
> ### Guidelines
> * [x]  I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

Yes I can help you
